data/rules.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.0"
```

Comment:
why did we need to add this one?

Reply:
- If you don't specify that, you get an `INFO` level message:
```
The 'version' key is missing in the training data file /Users/degiz/Work/rasahq/rasa-x-demo/data/rules.yml. Rasa Open Source will read the file as a version '3.0' file. See https://rasa.com/docs/rasa/training-data-format.
```
- This line exists in [original](https://github.com/RasaHQ/rasa/blob/3.0.x/rasa/cli/initial_project/data/rules.yml) `rules.yml` file, seemed appropriate to copy it over as it is
So overall having this like seems cleaner to me, and will nudge us to update the `rasa-x-demo` once the new minor OSS is released.

Reply:
Or do you mean why I added rules in the first place?

Reply:
So again, since the `README` explicitly specifies the initial rasa project as the source of truth, I copied it over.

Reply:
> Or do you mean why I added rules in the first place?

Yep 👍🏻 

> So again, since the README explicitly specifies the initial rasa project as the source of truth, I copied it over.


🚀